src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/Blocks/customer_preferences_general.html.twig

```diff
@@ -35,6 +35,17 @@
     </h3>
     <div class="card-body">
       <div class="form-wrapper">
+        <div class="alert alert-danger d-none" role="alert" id="configurationFormAlertMessageOptin">
+          <div class="alert-text">
+            <p>{{ 'Before disabling partner offers, please [1]make[/1] sure they are not set as required in the [2]Customers[/2] section of the back office. Otherwise, new customers won\'t be able to create an account and [1]proceed[/1] to checkout.'|trans({
```

Comment:
```suggestion
            <p>{{ '[1]Make[/1] sure partner offers are not set as required in the [2]Customers[/2] section of the back office before disabling them. Otherwise, new customers won\'t be able to create an account and [1]proceed[/1] to checkout.'|trans({
```

Reply:
Fixed